app/code/Magento/InventoryInStorePickupShippingApi/Model/Carrier/InStorePickup.php

```diff
@@ -153,7 +153,7 @@ private function createResultMethod(float $shippingPrice): Method
     /**
      * @inheritdoc
      */
-    public function getAllowedMethods()
+    public function getAllowedMethods(): array
```

Comment:
This method implements `\Magento\Shipping\Model\Carrier\CarrierInterface::getAllowedMethods` which does not have `array` type. It's best to remain consistent with interfaces.